.gitmodules

```diff
@@ -1,3 +1,11 @@
+# SPDX-License-Identifier: CC0-1.0
+
+# SPDX-FileCopyrightText: 2015 espes
+# SPDX-FileCopyrightText: 2016 Matt Borgerson
+# SPDX-FileCopyrightText: 2017 Luke Usher
+# SPDX-FileCopyrightText: 2018-2019 Stefan Schmidt
+# SPDX-FileCopyrightText: 2019 Jannik Vogel
```

Comment:
Does this file really need copyright info? Is there even anything non-factual that *could* potentially be copyrighted?

Reply:
Probably not, but that's part of the reason why I made it explicitly CC0, so that no-one will have to worry about that (as well as making `reuse lint` and similar tools happy).

Reply:
Agree license info isnt necessary here but happy to leave it for lint tools.
Where is this author list from? Commit history has different contributors than listed.

Reply:
Oops, that was the commit history of `.gitignore` (mixed it up in the commit title, too). Fixed it now, and added another commit that adds license and copyright info for `.gitignore`.